next-frontend/src/components/about/CallToAction.tsx

```diff
@@ -0,0 +1,44 @@
+"use client";
+
+import { Box, Button, Stack, VStack } from "@chakra-ui/react";
+import React from "react";
+import { MarkdownProse } from "@/components/Markdown";
+
+type CallToActionBlockProps = {
+  content: string;
+  buttons: {
+    label: string;
+    url: string;
+  }[];
+};
+
+export function CallToActionBlock({
+  content,
+  buttons,
+}: CallToActionBlockProps) {
+  return (
+    <Box borderRadius="2xl" p={{ base: 6, md: 10 }} shadow="md" width="full">
+      <VStack align="start">
+        <Box color="gray.700" fontSize="lg">
+          <MarkdownProse content={content} />
+        </Box>
+
+        <Stack direction={{ base: "column", sm: "row" }}>
+          {buttons.map((button, i) => (
+            <Button
+              key={i}
+              as="a"
+              colorScheme="blue"
+              variant={i === 0 ? "solid" : "outline"}
+              size="lg"
+            >
+              <a href={button.label} rel={"noopener"}>
+                {button.label}
+              </a>
```

Comment:
I don't think you need to use low-level `a` tags here manually. Both Chakra and NextJS have a `Link` component, and you should use Chakra's if it's an external link or NextJS' if it's an internal link to one of our own pages within the app

Reply:
You do, as per Button as Link documentation in https://chakra-ui.com/docs/components/button#as-link